### ISSUE TITLE
dot Net 9 runtime issue - IIS Proxy Install

### ISSUE THREAD

**Issue by bevanm8** [bagetter/BaGetter#202]
### Describe the bug
Might not be a bug, but the instructions on bagetter website link to the latest dot net which didn't work for me. IIS kept saying "missing version of ASP.NET". I installed the dot Net 8 version and the issue was resolved, the application loads correctly. If BaGetter requires version 8, the instructions should be updated to say this. (I don't know, I just know dot net 8 fixed my issue)
Guide link:
https://www.bagetter.com/docs/Installation/iis-proxy

### To Reproduce
1. Install ASP Net 9 Hosting version
2. Reboot machine
3. Extract BaGetter zip 
4. Set up App Pool as per guide
5. Create website as per guide using BaGetter app pool
6. Select BaGetter folder for Website
7. IIS Reset for good measure
8. Navigate to port 5000 on browser
9. Error: Missing ASP.NET version

### Expected behavior
The application should load on port 5000

**Comment by Regenhardt** [comment]
You're right, we should include the current framework version of BaGetter in the docs, thanks for raising this issue.
